### PR TITLE
[#223] Use MySQL ODBC connector 8.0.33 for MariaDB projects

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #223

I've left a boolean at the top of `odbc_setup.py` that can be used to switch back to the MariaDB ODBC driver, to allow for easier testing when working on irods/irods#7323.

Once irods/irods#7323 is fixed, this PR should be reverted, unless it has already been superseded by #208.

### CODE REVIEW

File: irods_testing_environment/odbc_setup.py [hunk 1]
```diff
@@ -450,13 +454,22 @@ def make_mariadb_odbcinst_ini(csp_container, container_odbc_lib_dir):
...
     execute.execute_command(csp_container, 'cat {}'.format(odbcinst_ini_path))
 
-def configure_mariadb_odbc_driver_apt(csp_container, odbc_driver):
+def configure_mariadb_odbc_driver_apt(csp_container, odbc_driver, package_url):
     """Configure ODBC driver package for mariadb (via apt)
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
+    package_url -- location of  ODBC driver package to download if odbc_driver is not provided
```
Comment (trel) [inline irods_testing_environment/odbc_setup.py:463]:
```suggestion
    package_url -- location of ODBC driver package to download if odbc_driver is not provided
```

File: irods_testing_environment/odbc_setup.py [hunk 2]
```diff
@@ -465,13 +478,22 @@ def configure_mariadb_odbc_driver_apt(csp_container, odbc_driver):
...
     make_mariadb_odbcinst_ini(csp_container, '/usr/lib/x86_64-linux-gnu')
 
-def configure_mariadb_odbc_driver_dnf(csp_container, odbc_driver):
+def configure_mariadb_odbc_driver_dnf(csp_container, odbc_driver, package_url):
     """Configure ODBC driver package for mariadb (via yum)
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
+    package_url -- location of  ODBC driver package to download if odbc_driver is not provided
```
Comment (trel) [inline irods_testing_environment/odbc_setup.py:487]:
```suggestion
    package_url -- location of ODBC driver package to download if odbc_driver is not provided
```

File: irods_testing_environment/odbc_setup.py (hunk 1 above)
Comment (SwooshyCueb) [reply irods_testing_environment/odbc_setup.py]: fixed

File: irods_testing_environment/odbc_setup.py (hunk 2 above)
Comment (SwooshyCueb) [reply irods_testing_environment/odbc_setup.py]: fixed

**Review by alanking — APPROVED** [review]
Seems good to me. Easy enough to switch back and forth. I say # it

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**